tests/fe/p1nc_01.output

```diff
@@ -0,0 +1,10 @@
+
+DEAL::index=0 position=0.56351 0.56351 values=0.63730 0.25000 0.25000 -0.13730 
```

Comment:
Yes. @bangerth told me to compute these values using my hands at first to compare.